src/legacy/core_plugins/kibana/public/discover/np_ready/angular/discover.js

```diff
@@ -85,6 +87,8 @@ const fetchStatuses = {
 
 const app = getAngularModule();
 
+app.factory('history', () => createHashHistory());
```

Comment:
It's a small thing, but as we are migrating away from angular, it would make sense to put the history object into `kibana_services` instead of an angular service - it's achieving the same thing (making sure there is only one `history` instance in discover), but we don't need to keep angular around for it.

Reply:
Makes sense.
Moved into `DiscoverServices`